### PR TITLE
Add new companies dataset

### PR DESCRIPTION
**Pull request by cuducos**
New data, as the previous was not so good, as [discussed here](https://github.com/datasciencebr/serenata-de-amor/issues/214).

### CODE REVIEW

File: serenata_toolbox/datasets/downloader.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ class Downloader:
         '2017-03-15-reimbursements.xz',
         '2017-03-20-purchase-suppliers.xz',
         '2017-04-27-companies-no-geolocation.xz',
```
Comment (jtemporal) [inline serenata_toolbox/datasets/downloader.py]: Shouldn't this be removed from this list in light of the new `companies-no-geolocation` dataset?

Comment (cuducos) [inline serenata_toolbox/datasets/downloader.py]: I'm not sure. If we think about versioning, this should be there. Maybe someone somewhere in the world is running an analysis using it. But I do agree it's a quiet odd possibility. So… being strict, no this should not be deleted… but being pragmatic, yes, let's get rid of that.

Comment (jtemporal) [inline serenata_toolbox/datasets/downloader.py]: I'm just saying the method is called `fetch_latest_backup` IMHO there should only be listed the most recent datasets. Am I wrong on assuming that? Maybe we should come up with a list of all the datasets available on the S3?

Comment (cuducos) [inline serenata_toolbox/datasets/downloader.py]: Make sense then!

Comment (cuducos) [inline serenata_toolbox/datasets/downloader.py]: Done in bbbf88b

**Pull request merged**